core/domain/rte_component_registry_test.py

```diff
@@ -203,17 +203,17 @@ def test_require_file_contains_all_imports(self):
 
         rtc_ts_filenames = []
         for component_id in feconf.ALLOWED_RTE_EXTENSIONS:
-            # TODO(#9356): Remove this if condition once the Svgeditor
-            # directive is created in the second part of 1st milestone.
-            if component_id == 'Svgeditor':
-                continue
             component_dir = os.path.join(
                 feconf.RTE_EXTENSIONS_DIR, component_id)
             directives_dir = os.path.join(component_dir, 'directives')
             directive_filenames = os.listdir(directives_dir)
+            # When reading for all the .ts files in the directives directory,
+            # the .spec.ts # files should not be included.
```

Comment:
Drop "#".

Also explain why (in the comment).

Reply:
It was a type when I tried to remove a line of comment. Done.